test/tt_importer_SUITE.erl

```diff
@@ -0,0 +1,19 @@
+-module(tt_importer_SUITE).
```

Comment:
@dktn have a look to to correctly store CT SUITE data: https://github.com/lambdaacademy/2015.12_erlang/commit/60dc366c90fe0a8f23da20707a02aa173b6d8f35